README.rst

```diff
@@ -58,3 +79,6 @@ You can test your doc changes locally with::
     $ cd docs
     $ make html
     $ open _build/html/index.html
+
+
+[service-dev-wiki]: https://github.com/yola/wiki/wiki/Service-and-Client-guidelines#development
```

Comment:
I agree. But I've been struggling with how to document the process of creating test credentials for this project. This was the best compromise I could come up with. Do you have any ideas? Or do you think it should just be communicated privately between Yola team members?